crates/ty_python_semantic/resources/mdtest/type_properties/is_fully_static.md

```diff
@@ -48,7 +48,9 @@ static_assert(not is_fully_static(Any | str))
 static_assert(not is_fully_static(str | Unknown))
 static_assert(not is_fully_static(Intersection[Any, Not[LiteralString]]))
 
-static_assert(not is_fully_static(tuple[Any, ...]))
+# TODO: should pass
```

Comment:
I guess in general we don't descend into generic instance types when detecting fully-staticness? Do we have an issue for this already? It seems important and not too hard to fix?